.github/workflows/cd.yaml

```diff
@@ -34,8 +34,21 @@ jobs:
         env:
            REGISTRATION_SERVICE_VERSION: 0.0.1-SNAPSHOT
 
+      - name: 'Copy Identity Hub CLI'
+        run: |
+          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.identityhub:identity-hub-cli:$IDENTITY_HUB_VERSION:jar:all -DoutputDirectory=.
```

Comment:
Copying dependencies using Maven will be reworked at: https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/issues/32